test/tests.py

```diff
@@ -990,6 +993,46 @@ def test(self):
                         dmlc_extraargs = ['--dep', 'T_dmldep.dmldep'],
                         api_version=latest_api_version))
 
+class DmlDepNinja(DMLFileTestCase):
```

Comment:
This looks like a modified copy of DmlDep. How much pain would it be to extend DmlDep with parameters to make that class cover both flags?